src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -2482,8 +2484,21 @@ def build_indexes(
         flow_custom_indexes
             List of custom indexes for the flows collection.
             Same as job_custom_indexes.
+        drop
+            If True all existing indexes in the collections will be dropped.
         """
+
+        if drop:
+            self.jobs.drop_indexes()
+            self.flows.drop_indexes()
+            self.auxiliary.drop_indexes()
+
         self.jobs.create_index("db_id", unique=True, background=background)
+        self.jobs.create_index(
```

Comment:
Actually this is the main point of the bug fix 😄 
I cannot put a "unique" index on uuid alone, because jobs with index>1 will have the same uuid. So the "unique" index that prevents having duplicated Jobs is this composed uuid+index.

Reply:
I see, I was only thinking about workflow creation time.